packages/react-storage/src/components/StorageBrowser/composables/types.ts

```diff
@@ -0,0 +1,11 @@
+import { StatusDisplay } from './StatusDisplay';
+
+const composables = {
+  StatusDisplay,
+};
+
+export type ComposableTypes = typeof composables;
+
+export interface Composables {
+  components?: ComposableTypes;
```

Comment:
Should it be `composables?: ComposableTypes`?

Reply:
I wasn't sure about this because I don't know what the final API for this would be. If we are exposing through createStorageBrowser, it will likely just be "components" publicly. The "composable" terminology is largely internal so that we can draw those lines of responsibility and organize thing